### ISSUE TITLE
Spinning up AWS requires curl

### ISSUE THREAD

**Issue by rca** [kubernetes/kubernetes#25075]
I tried running the command in the "Running Kubernetes on AWS EC2" docs with the `wget` version of the command since `curl` was not installed:

```
export KUBERNETES_PROVIDER=aws; wget -q -O - https://get.k8s.io | bash
```

The `aws` flavor, however does require `curl` to be installed, for example `wait-master` in `util.sh`: https://github.com/kubernetes/kubernetes/blob/release-1.2/cluster/aws/util.sh#L1267

Is a test for `curl` valid for adding into the `verify-prereqs()` function within `util.sh`?

**Comment by roberthbailey** [comment]
I bet that the GCE provider requires curl for the same reason.

**Comment by justinsb** [comment]
I think this will likely be fixed as part of #23478

**Comment by justinsb** [comment]
Closing as kube-up is in maintenance mode, and fixing this feels out of scope for maintenance.
